demos/node-classification/attri2vec/attri2vec-citeseer-node-classification-example.ipynb

```diff
@@ -104,6 +104,7 @@
       "\n",
```

Comment:
<p>Wow, so much faster!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
oh I didn't notice; is that from the new stellar class?

Reply:
I'm guessing so, each epoch has dropped to 4s from 30s!

Reply:
wow, thats amazing! nice work @kjun9 @huonw

Reply:
<p>Interesting. I'll investigate to verify, because if it is that much faster, that's a good-news-story we can tell!</p>

Reply:
Ah, unfortunately, I cannot reproduce this: I tried running on the latest develop (69fc13bd) and observed epochs taking ~26s, which is the same as what it took on the commit that switched over to the new StellarGraph (2dfff02d701922ed1f2164945dbd75651f9ab6a8), and on the commit before that (172edad0).

Maybe you're running this on a GPU, @kieranricardo?

Reply:
> Maybe you're running this on a GPU, @kieranricardo? Or a particularly powerful CPU? (Or some other reason for the difference?)
 
We suspect it may be because he installed `tensorflow-mkl`, and is thus getting the most out of his (Intel) CPU.